pkg/watch/mux.go

```diff
@@ -149,6 +151,7 @@ func (m *Broadcaster) Action(action EventType, obj runtime.Object) {
 // You must not call Action after calling Shutdown.
```

Comment:
You mean blocked channel sends because the receivers aren't listening yet? In that case, since our channel send has blocked waiting for a listener, the Done() call would not be done and we would wait. So all events must have been sent, nothing can remain sitting in outbound channels - at least with the current approach.